### PR TITLE
feat: job posting for executive assistant

### PR DESCRIPTION
**Pull request by marneysof**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by vinckr** [comment]
There was an extra `'` in the job description @marneysof, that is why the first check there failed.

**Comment by vinckr** [comment]
I also shortened the description a bit so it was roughly the same length as the others.

**Comment by jfcurran** [comment]
good stuff everyone, it reads very nicely 👀

**Comment by jfcurran** [comment]
@tacurran please review this pull-request :)

File: src/markdown/jobs/execassist.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+develop open-source software and use it to build a scalable, easy-to-use, and
+secure cloud platform. You will talk to customers to understand their biggest
+problems and tailor solutions to solve them, having a positive impact on
+hundreds of millions of people worldwide.
+
+### Overview
+
+- **Position type:** Full time
+- **Location**: Munich, Germany
+- **Components**: Google Suite, Github
```
Comment (aeneasr) [inline src/markdown/jobs/execassist.mdx:34]:
```suggestion
- **Components**: Google Suite, Github, Slack
```

File: src/markdown/jobs/execassist.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  create and meet efficient deadlines
+- Curiousity, creativity and an eagerness to develop new competencies as new
+  challenges arise
+- Experience in collaborative management with design, engineering, data and
+  analytics, marketing, and operations
+- 2+ years of experience in an assistance role, ideally at a software company
+  and/or startup
+- Familiarity with software development
+- Talent for understanding and explaining customer problems
+- Techical fluency with G Suite tools, Slack, Github
```
Comment (aeneasr) [inline src/markdown/jobs/execassist.mdx:70]: Nice to have but not required - I think very few EAs have GitHub experience :)

File: src/markdown/jobs/execassist.mdx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  communications on behalf of the CEO
+- A high level of organizational skills, attention to detail, and ability to
+  create and meet efficient deadlines
+- Curiousity, creativity and an eagerness to develop new competencies as new
+  challenges arise
+- Experience in collaborative management with design, engineering, data and
+  analytics, marketing, and operations
+- 2+ years of experience in an assistance role, ideally at a software company
+  and/or startup
+- Familiarity with software development
```
Comment (aeneasr) [inline src/markdown/jobs/execassist.mdx:68]: Not required for an EA - nice to have of course

File: src/markdown/jobs/execassist.mdx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+- Organize company events, e.g. the monthly All-Hands Meeting
+- Collaborate with the technical product management, engineering, sales and
+  marketing teams to facilitate well-rounded decision making with a bias for
+  action and calculated risks
+- Collaborate with Sales and Marketing to facilitate rapid time to value for our
+  customers and prospective customers
+
+### We’re looking for someone who has
+
+- Accomplished oral and written communication skills, ablity to write precise
```
Comment (aeneasr) [inline src/markdown/jobs/execassist.mdx:58]: fluent written english + german

**Comment by vinckr** [comment]
Implemented the suggestions from this PR review and added the current changes to /jobs in https://github.com/ory/web/pull/376.

**Pull request closed without merging**